pbrd/pbr_vty.c

```diff
@@ -73,7 +73,7 @@ DEFPY(pbr_map_match_vlan_id, pbr_map_match_vlan_id_cmd,
 
 /* clang-format off */
 DEFPY(pbr_map_match_vlan_tag, pbr_map_match_vlan_tag_cmd,
-      "[no] match vlan [<tagged|untagged|untagged-or-zero>$tag_type]",
+      "[no] match vlan ![<tagged|untagged|untagged-or-zero>$tag_type]",
```

Comment:
This does not do what you think it does.  Have you actually looked at the cli with this change?

Reply:
Could you please be more specific? I have looked at the CLI with this change but I'm not sure what you have in mind.

Reply:
adding a `!` doesn't do anything but require the cli to expect a `!` character

Please just remove this stuff and put an assert(tag_type) right before you call the sub function

Reply:
Is the following (from doc/developer/cli.rst) incorrect? It seems to behave as documented for me:

> ![exclamation square brackets] – same as [square brackets], but only allow skipping the contents if the command input starts with no. (For cases where the positive command needs a parameter, but the parameter is optional for the negative case.)